### DISCUSSION TITLE
Selective nuke

### DISCUSSION THREAD

**Discussion by inpendio** [fastlane/fastlane#19977] [Q&A] [answered]
Hi,
I have a situation like this:
- when the project was created there was only 1 app
- now there are multiple apps ( with their own profiles and certificates handled outside our fastlane setup)
As our certificates have expired I need to nuke old ones and create new ones. But fastlane only offers me to nuke everything. Since I don't want to nuke others teams certs this will not work for me.
So, is there any way to nuke only one cert and repair only a certain profiles?

**Comment by lucgrabowski** [comment, marked as answer]
You can select one certificate to be nuked if you're using the newest fastlane. You could read more about it in description of https://github.com/fastlane/fastlane/pull/19584
Don't use `force` = true, I'm saying just in case ... ;) And don't do it on CI.

**Reply by inpendio** [reply]
@lucgrabowski 
Looks like a have a bit of a different problem. I managed to delete certificates from the developer portal. They expired. Now, I have provisioning profiles without certificates.
I tried running `fastlane match development --force` but that creates new profiles. 
Running `fastlane sigh repair` throws an error ( on line where it checks the certificates) 
Any ideas about what could be done here? Should I nuke everything? I'm not much of an iOS dev, so I don't know if I nuke them, will I'll be able to repair the damage done ( there are several apps involved under our team and multiple profiles)

**Reply by lucgrabowski** [reply]
>Now, I have provisioning profiles without certificates

Those profiles are invalid now.

You could manually delete those profiles and recreate them with match.
or
You could manually edit those profiles and fix them in developer center but that way those profiles won't be in repository handled by match.

**Reply by inpendio** [reply]
Ok, tnx. Can be closed now

**Comment by jakub-przy** [comment]
I am sorry to dig this old thread up but I've been wondering... why should we NOT do it?

> And don't do it on CI.

Is it not possible technically, or just not advised? What are the downsides of managing fastlane_nuke automatically from the CI?

Cheers
